.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Report a bug in Bee
+about: Create a report to help us improve
+title: ""
+labels: c-bug
+---
+
+## Bug description
+
+Briefly describe the bug.
+
+## Rust version
+
+Which version of Rust are you running?
+
+- Rust version:
+
+## Bee version
+
+Which version of Bee are you using?
+
+- Bee version (version number, commit, or branch): 
+
+## Hardware specification
+
+What hardware are you using?
+
+- Operating system:
```

Comment:
OS:

Reply:
I think it's more inclusive to avoid acronyms. We might get some contributors whose first language isn't English.